inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -126,9 +131,29 @@ def name2marc(self, key, value):
 @utils.for_each_value
 def ids(self, key, value):
     """All identifiers, both internal and external."""
+    IDS_MAP = {
+        'ARXIV': 'ARXIV',
+        'BAI': 'INSPIRE BAI',
+        'CERN': 'CERN',
+        'DESY': 'DESY',
+        'INSPIRE': 'INSPIRE ID',
+        'KAKEN': 'KAKEN',
+        'ORCID': 'ORCID',
+        'SLAC': 'SLAC',
+        'WIKIPEDIA': 'WIKIPEDIA',
+    }
+
+    type_ = IDS_MAP.get(value.get('9').upper())
+    a_value = value.get('a')
+
+    if type_ == 'CERN' and a_value.startswith('CERN-'):
+        a_value = 'CERN-' + NON_DIGIT.sub('', a_value)
```

Comment:
I'm working around this guy from the demo records: https://github.com/inspirehep/inspire-next/pull/1428/files/6f35e65dabeeec73051df6266cc3e029358f6c4d#diff-4f22d81c877fcfe2a6d3ee287e0d3658R162